source/docs/software/roborio-info/roborio2-imaging.rst

```diff
@@ -0,0 +1,56 @@
+RoboRIO2 Imaging
+================
+
+The NI roboRIO 2.0 boots from a microSD card configured with an appropriate boot image containing the NI Linux Real-Time OS, drivers, and libraries specific to FRC. There are two ways to create the SD card -- with the roboRIO using the imaging tool, or with a laptop and an SD burner application.
+
+microSD Requirements
+--------------------
+
+The NI roboRIO 2.0 controller supports microSD cards.
+
+Operation Tips
+--------------
+
+The NI roboRIO 2.0 controller requires a fully inserted SD card containing a valid image in order to boot and operate as intended.
+
+If the SD card is removed while powered, the controller will hang. Once the SD card is replaced, the controller will need to be restarted using the reset button, or it can have the power cycled.
+
+No damage will result from SD card removal or insertion while powered, but best practice is to perform these operations while unpowered.
+
+Imaging Using a Computer
+------------------------
+
+The image will be transferred to the SD card using a specialized writing utility, sometimes called a burner. Several utilities are listed below, but most that can write arbitrary images for booting a Raspberry Pi or similar dev boards will also produce a bootable SD card for roboRIO 2.0.
+
+Supported image files are named FRC_roboRIO2_VERSION.img.zip and should be located next to this instruction file. You may also locate then by clicking the SD button in the roboRIO Imaging tool. Select the version that matches the version of the programming tools being used. A mismatch between tools and image may result in inconsistent library dependencies.
```

Comment:
Is there something we can copy from the other imaging article about selecting the right version? The current wording doesn't really help someone to determine what the right version is.

Reply:
I attached their article (remove .txt).  We have done more proofreading then they have.  I don't understand exactly what is meant here either as the current version is "2.3" but I don't see that version on any other "programming tools
![SD_Button](https://user-images.githubusercontent.com/15695521/142729894-b69b4001-b63e-452c-8a7e-4e2246d159b5.png)"
[Instructions for SD Imaging.html.txt](https://github.com/wpilibsuite/frc-docs/files/7574665/Instructions.for.SD.Imaging.html.txt)

Reply:
I have asked for clarification on this.

Reply:
The regular article just says choose the latest image, which is probably right 99% of the time.

Reply:
Do we want to just put that then?  I just wanted to understand this.